### DISCUSSION TITLE
Permission error after upgrade to v0.10.0

### DISCUSSION THREAD

**Discussion by rkranendonk** [google/docsy#2016] [Q&A]
Running `hugo server` generates the following error:

```
Error: error copying static files: open /Users/username/Documents/GitHub/canvasmethod-site/public/js/tabpane-persist.js: permission denied
```

Can I repair this?

Hugo version is v0.126.1

**Comment by deining** [comment]
Most likely not an error with docsy theme.
Nonetheless, can you give this a try, please:

```
hugo server --noChmod --noTimes
```

Does this cure your problem?

**Reply by rkranendonk** [reply]
No, that didn't solve it. 
If this is not an error with docsy theme, where should I continue my search for a solution, please?

**Reply by deining** [reply]
> No, that didn't solve it.

Hard to give more advice without knowing your environment. Seems like you are on linux correct?
Access rights for `tabpane-persist.js` seem to prevent copying or some process is holding a lock on the file to be  copied.

What you can try:

* Restart your computer and see if the problem persist
* Run `hugo mod clean` and see if the problem persist
* Try to identify your module cache, seek for `tabpane-persist.js` inside the cache and check access rights for this file.

**Comment by rkranendonk** [comment]
I'm on macOS Sonoma 14.5
Sorry to say that the first two suggestions didn't work.

A contributor on discourse.gohugo.io suggested deleting the public directory before building the site, that solved it.
(you could see that as a brute force way to fix the access rights for the file in question, I guess)

**Comment by chalin** [comment]
FYI, the following might be related:

- https://github.com/google/docsy-example/issues/217

If starting from a clean `public` folder fixed your issue, then I don't think that the problem was specific to Docsy 0.10.0. Btw, what were you upgrading from?

**Comment by rkranendonk** [comment]
The problem started after upgrading Hugo form 0.124.1 to 0.126.1. 
I can't remember what version of Docsy I was on, but if you can tell me how I can retrieve that I will be happy to provide.
